src/main/java/com/contentgrid/gateway/runtime/web/ContentGridAppRequestWebFilter.java

```diff
@@ -59,5 +56,26 @@ public Mono<Void> filter(ServerWebExchange exchange, WebFilterChain chain) {
                 .then(chain.filter(exchange));
     }
 
+    private Mono<ServiceInstance> logServiceInstanceNotFound(ServerWebExchange exchange) {
```

Comment:
```suggestion
    private Mono<?> logServiceInstanceNotFound(ServerWebExchange exchange) {
```

This never returns a ServiceInstance, so don't claim to return such a mono?

Reply:
But fine, resolved this another way.